galaxy/main/models/importing.py

```diff
@@ -18,10 +19,27 @@ class Meta:
     STATE_FAILED = constants.ImportTaskState.FAILED.value
     STATE_SUCCESS = constants.ImportTaskState.SUCCESS.value
 
+    # collection related
+    artifact = models.ForeignKey(
```

Comment:
How artifact reference in this model is planned to be used?

Reply:
@cutwater I wanted to associate the import task on creation to something related to collection. But haven't needed artifact reference and it is not persistent in database, I am thinking it is better to remove it